test/example_data.dart

```diff
@@ -220,6 +233,16 @@ Subscription subscription(
   );
 }
 
+UserTopicItem userTopicItem(
```

Comment:
test: Factor out `userTopicItem` helper

Is this commit intended to be NFC?
https://github.com/zulip/zulip-mobile/blob/main/docs/glossary.md#nfc

If so, it's helpful to the reader to mark it as such.